frontend/app/student/run-simulation/[instanceId]/page.tsx

```
@@ -1608,30 +1689,32 @@ ${availablePersonas.map(persona => `• @${persona.name.toLowerCase().replace(/\
                   onClick={() => setActiveTab('conversation')}
-                  className={`px-6 py-3 text-sm font-medium border-b-2 transition-colors ${
+                  className={`sim-tab px-6 py-3 text-sm font-medium border-b-2 ${
                     activeTab === 'conversation'
-                      ? 'border-blue-500 text-blue-600'
-                      : 'border-transparent text-gray-500 hover:text-gray-700'
+                      ? 'sim-tab-active text-blue-600 border-transparent'
+                      : 'border-transparent text-gray-500'
                   }`}
+                  style={{ fontFamily: "'Sora', sans-serif" }}
                 >
                   <MessageCircle className="w-4 h-4 mr-2 inline" />
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)